packages/core/src/components/ModalNew/Modal/Modal.types.tsx

```diff
@@ -0,0 +1,15 @@
+import { VibeComponentProps } from "../../../types";
+import React from "react";
+import { ModalTopActionsProps } from "../ModalTopActions/ModalTopActions.types";
+
+export type ModalSize = "small" | "medium" | "large";
+
+export interface ModalProps extends VibeComponentProps {
+  show: boolean;
+  size?: ModalSize;
+  closeButtonTheme?: ModalTopActionsProps["color"];
```

Comment:
@shlomitc suggests deciding on a different structure for this, that we would apply on all our components.
Right now the pattern is also "[prefix]Theme" for Tipseen for example. (also for close button theming).
let's discuss if we want to change it.

Reply:
Not sure I understand